smallfile_wrapper/trigger_smallfile.py

```diff
@@ -137,6 +141,23 @@ def emit_actions(self):
                             interval['99%'] = float(flds[10])
                             yield interval, '-rsptimes'
 
+            if self.clients > 1:
+                self.logger.info("Number of smallfile pods > 1, synchronizing them for the next operation")
```

Comment:
There is never a case we _don't_ want them to be synchronized?

Reply:
In case we perform the benchmark with only one client, we skip sync code.